src/main/java/seedu/hirehub/MainApp.java

```diff
@@ -113,7 +117,21 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
             initialJobs = new UniqueJobList();
         }
 
-        return new ModelManager(initialData, initialJobs, userPrefs);
+        Optional<UniqueApplicationList> applicationListOptional;
+        UniqueApplicationList initialApplications;
+        try {
+            applicationListOptional = storage.readApplicationList(initialJobs, initialData);
+            if (!applicationListOptional.isPresent()) {
+                logger.info("Creating a new data file " + storage.getApplicationFilePath()
+                    + " populated with a sample application list.");
+            }
+            initialApplications = applicationListOptional.orElseGet(SampleDataUtil::getSampleUniqueApplicationList);
+        } catch (DataLoadingException e) {
+            logger.warning("Data file at " + storage.getApplicationFilePath() + " could not be loaded."
+                + " Will be starting with an empty jobs list.");
```

Comment:
Should the comment be "starting with an empty applications list" instead?

Reply:
fixed